code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1472,7 +1472,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 					burn_damage = HEAT_DAMAGE_LEVEL_2
 		burn_damage *= heatmod
 		if((prob(burn_damage) * 10) / 4)	//40% for level 3 damage on humans
-			H.emote("scream")
+			if (H.stat != UNCONSCIOUS)
```

Comment:
Merge this into the above if

Reply:
thought I did but missed it, fixing.

Reply:
it's already merged into the above if, it's indented

Reply:
he means change the if above to read `if(H.stat != UNCONSCIOUS && (prob(burn_damage) * 10) / 4))`, also you should change the condition to be either `!H.stat` or `H.stat > UNCONSCIOUS`, the former is a fairly generic "is not dead, unconsciout or in crit" check, the latter allows for soft crit as well as conscious.

Reply:
thanks! fixing it now. i'm pretty new to BYOND coding and especially to checking human properties. not familiar with stat, but familiar with if operators.